### DISCUSSION TITLE
请问 1.x 怎么获得群员头像？

### DISCUSSION THREAD

**Discussion by Arondight** [takayama-lily/oicq#256] [Q&A] [answered]
看了下代码没发现可用的属性 @takayama-lily

**Comment by JeffGe05** [comment, marked as answer]
这个使用 Web 方式获得：[web-api.md](https://github.com/takayama-lily/oicq/blob/main/web-api.md)

**Reply by Arondight** [reply]
感谢，可以了，另外问一下，那个 size（默认是0）具体是啥，我试了100 和 1000 好像没用

**Reply by JeffGe05** [reply]
0 原图
1, 2, 3, 4, 5 尺寸压缩后的头像，从小到大

**Reply by Arondight** [reply]
感谢老大哥！
